dd-sdk-android/src/main/kotlin/com/datadog/android/webview/DatadogEventBridge.kt

```diff
@@ -96,6 +98,35 @@ internal constructor(
     // endregion
 
     companion object {
+
+        internal const val JAVA_SCRIPT_NOT_ENABLED_WARNING_MESSAGE =
+            "You are trying to enable the WebView" +
+                "tracking but the java script capability was not enabled for the given WebView."
+        internal const val DATADOG_EVENT_BRIDGE_NAME = "DatadogEventBridge"
+
+        /**
+         * Will attach the [DatadogEventBridge] which will enable the RUM feature for the WebView.
+         * @param webView on which to attach the bridge
+         * Please note that:
+         * - you need to enable the java script support in the WebView settings for this feature
+         * to be functional:
+         * ```
+         * webView.settings.javaScriptEnabled = true
+         * ```
+         * - to make sure that all the navigation stays inside the [WebView] allowing us to track
+         * all the events, a [android.webkit.WebViewClient] needs to be attached:
+         * ```
+         * webView.webViewClient = WebViewClient()
+         * ```
+         * [More here](https://developer.android.com/guide/webapps/webview#HandlingNavigation).
```

Comment:
```suggestion
         * Attach the [DatadogEventBridge] to track events from the WebView as part of the same session.
         * Please note that:
         * - you need to enable the JavaScript support in the WebView settings for this feature
         * to be functional:
         * ```
         * webView.settings.javaScriptEnabled = true
         * ```
         * - by default, navigation will happen outside of your application (in a browser or a different app). To prevent that and ensure Datadog can track the full WebView user journey, attach a [android.webkit.WebViewClient] to your WebView, as follow:
         * ```
         * webView.webViewClient = WebViewClient()
         * ```
         * @param webView the webView on which to attach the bridge
         * [More here](https://developer.android.com/guide/webapps/webview#HandlingNavigation).
```